src/main/resources/templates/courses/assignments/group-assignment-view.ftl

```diff
@@ -0,0 +1,169 @@
+
+            [#if assignment.getDueDate()??]
+                <p>
+                    <strong>Due date</strong>
+                    <span>${assignment.getDueDate()}</span>
+                </p>
+            [/#if]
+
+            <table class="table table-bordered">
+            [#if myDeliveries?? && myDeliveries?has_content]
+                [#list myDeliveries as delivery]
+                    [#assign commitId = delivery.getCommitId()!]
+                    [@commitRow.render group states commitId! ""]
+                        [@deliveryStateButton user course delivery/]
+
+                        <div class="committer">${delivery.createdUser.getName()} on ${delivery.getCreated()?string["EEEE dd MMMM yyyy HH:mm"]}</div>
+
+                        [#if delivery.getNotes()??]
+                            <p>${delivery.getNotes()}</p>
+                        [/#if]
+
+                        [#assign attachments = delivery.getAttachments()!]
+                        [#if attachments?has_content]
+                            <ul class="list-inline">
+                            [#list attachments as attachment]
+                                <li>
+                                <a class="btn btn-link btn-sm" target="_blank" href="/courses/${group.course.getCode()}/groups/${group.getGroupNumber()}/assignments/${assignment.getAssignmentId()}/attachment/${attachment.getPath()?url('ISO-8859-1')}">
+                                    <span class="glyphicon glyphicon-file aria-hidden="true"></span>
+                                ${attachment.getFileName()}
+                                </a>
+                                </li>
+                            [/#list]
+                            </ul>
+                        [/#if]
+
+                        [#assign review = delivery.getReview()!]
+                        [#if review?? && review?has_content && (review.getGrade()?? || review.getCommentary()??)]
+                            <blockquote>
+                                <dl>
+                                    [#if review.getGrade()??]
+                                        <dt>Grade</dt>
+                                        <dd>${review.getGrade()}</dd>
+                                    [/#if]
+                                    [#if review.getCommentary()??]
+                                        <dt>Remarks</dt>
+                                        <dd>${review.getCommentary()}</dd>
+                                    [/#if]
+                                </dl>
+                                <footer class="small">${review.reviewUser.getName()} on ${review.getReviewTime()?string["EEEE dd MMMM yyyy HH:mm"]}</footer>
+                            </blockquote>
+                        [/#if]
+                    [/@commitRow.render]
+                [/#list]
+            [#else]
+                <tr>
+                    <td>
+                        No documents have been uploaded yet.
```

Comment:
Since submissions can be done without providing documents (only a commit), a proposed change:

"This group has not made a submission for this assignment yet."